transcom/mymove
# B20446-INT-payment requests doc viewer

## Agility ticket
[B20446](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20446)

## Summary
When a TIO is on the "Payment Requests" tab, there will be a list of payment requests.  Currently, for requests that have been approved or rejected, there will be a "View Documents" link that goes to the page to view the details.  A change was made that when the payment request has no documents, "No documents provided" will be displayed instead of "View Documents".  If the request has documents, the "View Documents" link will appear.

### How to test
* Customer: Create a move with an HHG shipment.
* SC: Approve move.
* TOO: Approve move.
* Prime:
  - Update shipment destination
  - Update shipment dates
    * Schedule Pickup Date
    * Actual Pickup Date
    * Schedule Delivery Date
  - Update shipment weights
    * Estimate Weight
    * Actual Weight
  - Create payment request with no documents
    * Create payment request
    * Do NOT upload any docs for the payment request
  - Create payment request with documents
    * Create payment request
    * Upload document to the payment request
* Prime
  - 1st payment request: Review and reject
  - 2nd payment request: Review and accept
  - Verification
    * 1st payment request: Confirm it has "No documents provided" text
    * 2nd payment request: Confirm it has "View Documents" link

## ANDI Screenshots
![image](https://github.com/user-attachments/assets/14c6545b-27fc-4106-8884-3b59b88ab700)

![image](https://github.com/user-attachments/assets/d831139e-4f29-444c-8fb3-e93b52f5ceff)
Only new changes have to be addressed
"View Documents" was pre-existing.
"No documents provided" has no action item so there's nothing to do for it.

## Testing Screenshots
Payment Request w/No Documents:
![image](https://github.com/user-attachments/assets/48c94c02-3ae5-4be8-b8bd-c8bb4205e144)


--------------------------------------------------------------------------
Payment Request w/Documents:
![image](https://github.com/user-attachments/assets/1d7733d0-65aa-47f6-bddd-f6066d7831e1)
